datasets/case03/queries/0001_showColumnsFromSource.sql.FIXME

```diff
@@ -1,3 +1,3 @@
 -- https://dev.lsstcorp.org/trac/wiki/dbPipeQAQueries 
 
-SHOW COLUMNS FROM DeepSource;
+SHOW COLUMNS FROM {DBTAG_A}DeepSource;
```

Comment:
These tests are marked FIXME, why add the database name to these tests, especially since they do not get run?

Reply:
It was the first query in the test, which if I was going to test lazy loading of databases, it would need to happen at the beginning of the test due to the proxy. I didn't notice the FIXME.

Reply:
Also, I deleted the {DBTAG_A} lines from the FIXME files. I'll add them if needed later.